AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.mm

```diff
@@ -13,6 +13,8 @@
 #import <list>
 #import <map>
 
+NSUInteger const ASDefaultTransactionPriority = 128;
```

Comment:
Maybe we should type this as an NSInteger rather than NSUInteger, especially because we don't really want to expose / have peolpe depend on the base value of 128.

Also then, for debugging purposes, we might as well have the base value be 0 perhaps?  or maybe 10, 100, some other human-percieved-round value.

Once this is in and tested, I am definitely interested in boosting the priority of ASTextNode drawing above images.  The feature you included to always have one thread running in queue order is a very good / important idea.